src/components/ZetkinDateTime.tsx

```diff
@@ -1,21 +1,25 @@
 import { FormattedDate, FormattedTime } from 'react-intl';
 
 interface ZetkinDateTimeProps {
+  convertToLocal?: boolean;
   datetime: string; // iso datetime string
 }
 
 const ZetkinDateTime: React.FunctionComponent<ZetkinDateTimeProps> = ({
+  convertToLocal,
   datetime,
 }) => {
   return (
     <>
       <FormattedDate
         day="numeric"
         month="long"
-        value={datetime}
+        value={convertToLocal ? new Date(datetime + 'Z') : datetime}
         year="numeric"
       />{' '}
-      <FormattedTime value={datetime} />
+      <FormattedTime
+        value={convertToLocal ? new Date(datetime + 'Z') : datetime}
+      />
```

Comment:
One thing i would do is move this turnary out to a variable so it's just in one place. Otherwise, it's good :)